crates/tasks_ui/src/lib.rs

```diff
@@ -113,9 +113,13 @@ fn task_context(
                         .map(|worktree| worktree.read(cx).abs_path().to_string_lossy().to_string())
                 });
 
+                let selected_text: String =
```

Comment:
I believe you don't need to state the type explicitly here, as that variable is used in a context where only `String` will suffice; let's give it a try and see if rustc can indeed infer it correctly;
```suggestion
                let selected_text =
```

Reply:
Letting rustc infer the type worked correctly. I had the type explicitly stated because I was testing the code with `dbg!()`, and rust couldn't infer it's type correctly.

I'm having trouble fix the task_ui test right now, once I do I'll update this merge request.